wordle-web/src/App.vue

```diff
@@ -25,4 +86,61 @@ function setInverseTheme() {
 function setDarkTheme() {
   theme.global.name.value = 'dark'
 }
+
+</script> -->
+
+<!-- <script>
+  export default {
+    data: () => ({
+      drawer: false,
+      group: null,
+      items: [
+        {
+          title: 'Home',
+          value: 'Home',
+        },
+        {
+          title: 'Better then Wordle',
+          value: 'Better then Wordle',
+        },
+        {
+          title: 'About',
+          value: 'About',
+        },
+      
+      ],
+    }),
+
+    watch: {
+      group () {
+        this.drawer = false
+      },
+    },
+  }
+</script> -->
+
```

Comment:
Not really an issue, but everything above this line should probably be removed unless you are actively using it as a reference.  It just makes the code a bit more difficult to read when the program doesn't start until line 122.